nbgrader/converters/base.py

```diff
@@ -19,6 +19,8 @@
 from ..nbgraderformat import SchemaTooOldError, SchemaTooNewError
 import typing
 from nbconvert.exporters.exporter import ResourcesDict
+from ipython_genutils.importstring import import_item
```

Comment:
I believe genutils isn't officially supported, and it would probably be better to not need to add a new dependency in any case---could you achieve the same behavior using `importlib.import_module`? See https://docs.python.org/3/library/importlib.html